### PR TITLE
add current dev issues note

### PR DESCRIPTION
**Pull request by telephon**
This would help those who haven't followed quite as closely.

### CODE REVIEW

File: README_CURRENT_ISSUES.scd [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+current issues
+
+(what you should know if you didn't follow the messages passed around)
+
+edit this file as needed to communicate the state of affairs.
+
+*/
+
+
+3.7 is basically done now.
```
Comment (crucialfelix) [inline README_CURRENT_ISSUES.scd]: this is a bit imprecise. it makes it sound like work is mostly finished and it should be released soon.  but what you mean is: we are doing maintenance and major bug fixes on it.  we have to discuss those things a bit anyway

**Comment by danstowell** [comment]
Honourable motives, but imho this is likely to be a repository of out-of-date info. (The assertion about 3.8's release date might be out of date already?) Feels to me like the ever-tempting MacGuffin of a "living document" that obviates the need for conversation...?

(Feel free to ignore my bah humbuggery)

**Comment by bagong** [comment]
Readme's are little read, and this would be the third now... Maybe a webpage with that purpose could be done? But simply scanning the first 10 issues is probably the most informative, and that list dynamically updated ;)

**Comment by telephon** [comment]
yes, you are all saying the correct thing. I just sometimes feel lost even though I check what happens every day. Many solutions to current problems are lost, and i simply can't find them anymore in the depths of logs. Wikis are better, so I suppose we could use the wiki for that. But also wikis are never updated unless they are really used.

**Comment by LFSaw** [comment]
maybe adding a date to "current issues" prevents upcoming issues with outdated information?

**Comment by danstowell** [comment]
Yeah Julian I think you're not the only one feeling that way.

I realised that this "note" file would also have different content depending on which branch you were on! Could get awkward. If it was a wiki page it wouldn't be tied to the branch, so at least that's one motivation for wiki...

**Comment by telephon** [comment]
@danstowell yes, you are completely right. https://youtu.be/1mbUdsQfSq0?t=38s

**Comment by bagong** [comment]
I think the difficulty is to make the Wiki being used. If it were used it would be updated. There are so many sources for current info on SC, and everybody has their own habits....
Maybe adding two or three tags of the type "sticky", "current problem", "attention" or so could make querying the issues + PRs more helpful? Once it's outdated people would remove it in their own interest.

**Comment by crucialfelix** [comment]
I agree with everybody else here.  we should use the normal wikipages, not the site one. that is easy enough to edit and contribute to.

github.io site should be just the front pages, intro to supercollider, release announcements.

**Comment by crucialfelix** [comment]
Consensus is to close.

**Pull request closed without merging**